MekHQ/src/mekhq/campaign/finances/Loan.java

```diff
@@ -594,4 +595,18 @@ public boolean equals(Object obj) {
                 && this.getRemainingPayments() == loan.getRemainingPayments()
                 && this.getRemainingValue() == loan.getRemainingValue();
     }
-}
\ No newline at end of file
+
+    @Override
+    public int hashCode() {
```

Comment:
For my education, what's the value of overriding the hashCode object if we don't explicitly need to?

Reply:
Two objects that are equal should have the same hash. Hash data structures may depend on that behavior for equality checks (that’s the case in C#, not sure in Java).

Reply:
Gotcha.